spec.html

```diff
@@ -22006,6 +22006,17 @@ <h1>Source Text Module Records</h1>
                 Meaning
               </th>
             </tr>
+            <tr>
+              <td>
+                [[Context]]
```

Comment:
I think it would make more sense to put the table-row for `[[Context]]` after the one for `[[ECMAScriptCode]]`, because `[[ECMAScriptCode]]` is kind of the defining characteristic of a Source Text Module, and `[[Context]]` arises from it.

Also, in ParseModule, the new STMR should presumably have a field for `[[Context]]`.